.github/workflows/c3-e2e-dependabot.yml

```diff
@@ -0,0 +1,86 @@
+# Runs the C3 e2e tests on dependabot PRs
+
+name: C3 E2E Tests (Dependabot)
+
+on:
+  pull_request:
+    paths:
+      - packages/create-cloudflare/**
+
+env:
+  node-version: 18.17.1
+  bun-version: 1.0.3
+
+jobs:
+  get-dependabot-bumped-framework:
+    name: "Get bumped framework (dependabot-only)"
+    runs-on: ubuntu-latest
+    outputs:
+      bumped-framework-cli: ${{ steps.detect.outputs.result }}
+    if: |
+      github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+      - name: Get PR description as string
+        id: get-pr-description
+        run: |
+          str=$(sed 's/`/\`/g' <<EOF
+            ${{ github.event.pull_request.body }}
+          EOF
```

Comment:
I have no idea what this does 😅

Reply:
this is already existing code just getting moved around 🙂 

anyways it to get the pull request description (that dependabot generates) and put it in this `str` variable

I had to implement it this was because the description contains multiple lines and since it gets interpolated it into the bash script itself I couldn't use a simple echo and similar